products/openzfs.md

```diff
@@ -0,0 +1,53 @@
+---
+permalink: /openzfs
+layout: post
+title: OpenZFS
+category: lang
+command: cat /sys/module/zfs/version
```

Comment:
Does this work on FreeBSD as well?

Reply:
I believe it does but it opened the possibility of the system package being up to date while the zpool not being up to date. Best to check the zpool version.